bug(?) website: " No loader is configured for ".node" files: node_modules/playwright/node_modules/fsevents/fsevents.node"

If I clone a fresh copy of the repo and run `nvm use 20` `npm i` `npm run start`, I get this message:

```
> pathoplexus-website@0.0.1 start
> astro dev

  🚀  astro  v3.3.0 started in 293ms

  ┃ Local    http://localhost:3001/
  ┃ Network  http://10.43.175.112:3001/
             http://10.1.112.211:3001/

✘ [ERROR] No loader is configured for ".node" files: node_modules/playwright/node_modules/fsevents/fsevents.node

    node_modules/playwright/node_modules/fsevents/fsevents.js:13:23:
      13 │ const Native = require("./fsevents.node");
         ╵                        ~~~~~~~~~~~~~~~~~
```

If I try to access e.g. the status page the process crashes with a similar message.

--- a/website/astro.config.mjs
+++ b/website/astro.config.mjs
@@ -19,6 +19,5 @@ export default defineConfig({
         optimizeDeps: {
             exclude: ['fsevents'],
         },
-    
     },
 });

